ultravox/training/train.py

```diff
@@ -129,10 +116,9 @@ def train(args: config_base.TrainConfig):
 
     logging.info("Instantiating model...")
 
-    # Since the model downloads the language model and audio encoder weights, we want one process to finish up
-    # downloading before the others start in order to avoid race conditions.
-    with ddp_utils.run_on_master_first(is_master):
-        model = ultravox_model.UltravoxModel(config)
+    # We assume that the weights are already downloaded via prefetch_weights.py
+    # If the weights are not downloaded, we might see a race condition here when using DDP.
+    model = ultravox_model.UltravoxModel(config)
```

Comment:
We could still keep `run_on_master_first` here to be safe (since we manually only download the `.safetensors` files and that can potentially be insufficient). But since that would've made the start time slower, I decided the added safety wasn't worth it.

Reply:
Actually never mind, I added a `.from_pretrained` call to cover that case too.